examples/autoscaling.yaml

```diff
@@ -32,11 +32,20 @@ SenzaComponents:
         # default instance count is 1
         # default timeout is 15 minutes
         SuccessRequires: "2 within 5m"
-        # scale two instances up/down at a time
+        # scale two instances up/down at a time (if not overwritten by scale up/down specififc value)
         ScalingAdjustment: 2
+        # scale three instances up at a time (if set, it overwrites the ScalingAdjustment value for scaling up)
+        ScalingUpAdjustment: 3
```

Comment:
I am a bit unhappy abut the name. We now have `ScaleUpThreshold` and `ScaleUpCooldown` but `ScalingUpAdjustment`. I chose that name nevertheless because the existing value is already called `ScalingAdjustment`. Same applies to the scale down pendant.

Any suggestions for more consistent names?